### PR TITLE
Fix warning for `.xor.`

### LINKED ISSUE #1136: Change warning for `.xor.`
Currently LFortran prints if you use `.xor.`:
```
 warning: .xor. is an LFortran extension
    --> ~/tools/lfortran/integration_tests/logical3.f90:74:5
     |
 74 | if (a .xor. b) then
     |     ^^^^^^^^^ please help us: what is the Fortran way to specify xor?
```
We should change it so that instead of `please help us: what is the Fortran way to specify xor?` it just says `LFortran extension`. I think then it would be clear and long term. And later we will have different modes, such as strict "standard conformance", vs "extensions", so we can turn some of these into errors, as needed based on the conformance "profile".

### PR DESCRIPTION
**Pull request by Pranavchiku**
Fixes #1136.

### CODE REVIEW

File: src/lfortran/semantics/ast_common_visitor.h [hunk 1]
```diff
@@ -359,7 +359,7 @@ class CommonVisitorMethods {
         diag.semantic_warning_label(
             ".xor. is an LFortran extension",
             {x.base.base.loc},
-            "please help us: what is the Fortran way to specify xor?"
+            ""
```
Comment (certik) [inline src/lfortran/semantics/ast_common_visitor.h:362]: Maybe put this in, that way there is some text next to the line.
```suggestion
            "LFortran extension"
```
My eyes usually look at the line first, so if the short text is self explanatory, then that's all I need to know.

Comment (Pranavchiku) [reply src/lfortran/semantics/ast_common_visitor.h:362]: Okay sure!

**Review by certik — APPROVED** [review]
I think this looks good now. I assume the removed test was a left over from some other PR that is not used as a reference anymore?

**Comment by Pranavchiku** [comment]
I guess yes, I am not sure though why it is removed, I just did `rm -rf ./tests/reference/*` and tthen `./run_tests.py -u`. It might be because of some other PR.

**Pull request merged**